renderdoc/driver/vulkan/vk_resources.cpp

```diff
@@ -1702,6 +1702,351 @@ uint32_t GetPlaneByteSize(uint32_t Width, uint32_t Height, uint32_t Depth, VkFor
   return GetByteSize(Width, Height, Depth, Format, mip);
 }
 
+struct TexelShape
```

Comment:
I think `TexelShape` is a misleading name. even for block formats a texel is a texel - it's just that NxM texels are encoded in a block. This really describes the _block_ shape where we consider non-block formats as a degenerate case with 1x1 blocks - so maybe `BlockShape` with a comment indicating such?

Reply:
Yes, I was temporarily confused about terminology. Done.